python/cudf_polars/cudf_polars/utils/config.py

```diff
@@ -99,6 +101,49 @@ def __post_init__(self) -> None:
             raise TypeError("pass_read_limit must be an int")
 
 
+def default_blocksize(scheduler: str) -> int:
+    """Return the default blocksize."""
+    try:
+        # Use PyNVML to find the worker device size.
+        import pynvml
```

Comment:
Can we rely on pynvml being available? I don't see it in our pyproject.toml: https://github.com/rapidsai/cudf/blob/1795de73cd84734609e475b60bc71afd25c649a1/python/cudf_polars/pyproject.toml#L23

maybe we should include `ImportError` in the list of exceptions?